fendermint/testing/graph-test/subgraph/graph-node/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node:v0.27.0
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
```

Comment:
Can you add more details for Readme about these IPFS and Postgres dependencies? It's not clear to me what's their role (if there's a link to any tutorial/post you followed, probably just linking is enough)

Reply:
This is what we followed: https://docs.hedera.com/hedera/tutorials/smart-contracts/deploy-a-subgraph-using-the-graph-and-json-rpc

And this is where The Graph itself documents its docker setup:https://github.com/graphprotocol/graph-node/blob/master/docker/README.md

Reply:
Great, i'd suggest to mention these 2 links in a comment.

Reply:
I added these links into README.md